src/rustup.ts

```diff
@@ -78,9 +78,16 @@ async function hasToolchain(): Promise<boolean> {
 async function tryToInstallToolchain(): Promise<void> {
     startSpinner('RLS', 'Installing toolchain…');
     try {
-        const { stdout, stderr } = await execChildProcess(CONFIGURATION.rustupPath + ' toolchain install ' + CONFIGURATION.channel);
-        console.log(stdout);
-        console.log(stderr);
+        const channel = window.createOutputChannel('Rustup');
+        channel.show();
+        // TODO: See if I need to split this into the path and the arguments
+        // TODO: Consider reporting error lines to the Problems pane?
```

Comment:
I don't think we should do that since they are kind of meta-errors and don't have a line number in the project, etc. We could surface them as pop-up warnings/errors (like we do with overall success).

Reply:
I've updated the code to instead collect the error separately in addition to outputting in in the Output channel and to display a window error notification message with the contents of the error, if any.